casbin/enforcer.h

```diff
@@ -147,9 +147,17 @@ class Enforcer : public IEnforcer{
         void BuildIncrementalRoleLinks(policy_op op, string p_type, vector<vector<string>> rules);
         // Enforce decides whether a "subject" can access a "object" with the operation "action", input parameters are usually: (sub, obj, act).
         bool Enforce(Scope scope);
+        // Enforce with two params, decides whether a "subject" can do the operation "action", input parameters are usually: (sub, act).
+        bool Enforce(string sub, string act);

```

Comment:
I think you should make an overloading function to be able to add a `matcher` argument to this function.
`bool Enforce(string matcher, string sub, string act)`
And there's a common problem among this overloading functions, that if we use the above function, it's going to get ambiguous with: 
`bool Enforce(string sub, string obj, string act);`

Reply:
> I think you should make an overloading function to be able to add a `matcher` argument to this function.
> `bool Enforce(string matcher, string sub, string act)`
> And there's a common problem among this overloading functions, that if we use the above function, it's going to get ambiguous with:
> `bool Enforce(string sub, string obj, string act);`

In the casbin-go,if we use Enforce(sub,obj,act),the matcher must be null.If we need to use matcher in casbin-go,we must use the Enforce with vector.
I think we should be in accordance with the go version.